km_api/know_me/tests/serializers/test_emergency_contact_serializer.py

```diff
@@ -35,14 +35,22 @@ def test_serialize(api_rf, emergency_contact_factory, serializer_context):
 
     url_request = api_rf.get(emergencycontact.get_absolute_url())
 
+    serializer_request = serializer_context['request']
+    has_read = emergencycontact.has_object_read_permission(serializer_request)
+    haswrite = emergencycontact.has_object_write_permission(serializer_request)
```

Comment:
This is a fine way of splitting it up. The only thing I would change is `haswrite` to `has_write` to match `has_read`.